### DISCUSSION TITLE
surface‘s touchscreen issues

### DISCUSSION THREAD

**Discussion by mojochean** [sumatrapdfreader/sumatrapdf#2511] [Q&A] [answered]
I want to swipe a pdf with my finger touch to turn pages on my surface. How can I do this?

**Comment by GitHubRulesOK** [comment, marked as answer]
"turn" pages does not apply to PDF viewers they "usually" scroll vertically which should work on a touch screen same as touch pad. You can rotate all pages from portrait/landscape to landscape/portrait mode if you need to, but the menu may then be at the side.

Those viewers that look like they turn pages on the web are not true document viewers. the page is an image thats warped to appear to be turning.

**Reply by mojochean** [reply]
Thanks for replying! Yes I want to vertically slide my PDF. I set the layout mode as "Continuous", but I can only scroll the PDF by my mouse as there is no feedback on the touching behaviour of my fingers.

**Reply by GitHubRulesOK** [reply]
Ok that should be standard behaviour but not having used my tablet for a while unsure of current behaviour so changes can only be added to current [pre-release](https://www.sumatrapdfreader.org/prerelease) make sure it is 64bit if your kit is new and **portable x64** should do, no need to install to test.

**Reply by mojochean** [reply]
thanks a lot
